builder/frameworks/picosdk.py

```diff
@@ -32,9 +32,10 @@ def gen_config_autogen(target_base_dir, board_hdr):
         env.Exit(-1)
     autogen_content_paths = [
         join(FRAMEWORK_DIR, "src", "boards", "include", "boards", board_hdr),
+    ]
+    if mcu == "rp2040":
         # only for ARM based RP2040
         join(FRAMEWORK_DIR, "src" , "rp2_common", "cmsis", "include", "cmsis", "rename_exceptions.h")
```

Comment:
This expression is standalone. Its result should still be added to the `autogen_content_paths` list.

Reply:
I see, I'll remove the if statement in a next commit (I have one question before pushing further below).  Doesn't seem to affect RP2350 compilation either way.

Reply:
Added now.